Description: Set transaction dbus no timeout.
 The libqapt.so transaction dbus interface uses default timeouts,
 This may cause the frontend (using libqapt.so) to timeout and
 return failure. But the backend (qaptworker3) is still waiting
 for polkit authorization.
Bug: https://pms.uniontech.com/bug-view-232297.html
Forwarded: not-needed
Last-Update: 2023-12-06

---
 src/transaction.cpp |    2 ++
 1 file changed, 2 insertions(+)

--- a/src/transaction.cpp
+++ b/src/transaction.cpp
@@ -49,6 +49,8 @@
             dbus = new TransactionInterface(QLatin1String(s_workerReverseDomainName),
                                             tid, QDBusConnection::systemBus(),
                                             0);
+            // dbus timeout, set no timeout.
+            dbus->setTimeout(INT_MAX);
         }
 
         ~TransactionPrivate()
